code/Maximum sum increasing.cpp subsequence

```diff
@@ -0,0 +1,47 @@
+/* Dynamic Programming implementation
+of Maximum Sum Increasing Subsequence
+(MSIS) problem */
+#include <bits/stdc++.h>
+using namespace std;
+
+/* maxSumIS() returns the maximum
+sum of increasing subsequence
+in arr[] of size n */
+int maxSumIS(int arr[], int n)
+{
+	int i, j, max = 0;
+	int msis[n];
+
+	/* Initialize msis values
+	for all indexes */
+	for ( i = 0; i < n; i++ )
+		msis[i] = arr[i];
+
+	/* Compute maximum sum values
+	in bottom up manner */
+	for ( i = 1; i < n; i++ )
+		for ( j = 0; j < i; j++ )
+			if (arr[i] > arr[j] &&
+				msis[i] < msis[j] + arr[i])
+				msis[i] = msis[j] + arr[i];
+
+	/* Pick maximum of
+	all msis values */
+	for ( i = 0; i < n; i++ )
+		if ( max < msis[i] )
```

Comment:
it is same as ```max(max, msis[i]);```
so why not use this simple function of code to reduce lines of code?